simulation_bridge/src/core/bridge_core.py

```diff
@@ -213,14 +237,17 @@ def _publish_message(self, producer, consumer, message,  # pylint: disable=too-m
             self.channel.basic_publish(
                 exchange=exchange,
                 routing_key=routing_key,
-                body=json.dumps(message),
+                body=json.dumps(message, default=datetime_serializer),
                 properties=pika.BasicProperties(
                     delivery_mode=2,
                 )
             )
             logger.debug(
                 "Message routed to exchange '%s': %s -> %s, protocol=%s",
                 exchange, producer, consumer, protocol)
+            # Record sent input time in performance monitor
+            if exchange == 'ex.bridge.output':
```

Comment:
I might have missed it before, but wouldn't the queue name be hard coded into the code base?